openedx/adg/lms/applications/models.py

```diff
@@ -149,21 +149,14 @@ class UserApplication(TimeStampedModel):
 
     organization = models.CharField(verbose_name=_('Organization'), max_length=255, blank=True, )
     linkedin_url = models.URLField(verbose_name=_('LinkedIn URL'), max_length=255, blank=True, )
-    resume = models.FileField(
-        max_length=500, blank=True, null=True, upload_to='files/resume/', verbose_name=_('Resume File'),
-        validators=[FileExtensionValidator(['pdf', 'doc', 'jpg', 'png'])],
-        help_text=_('Accepted extensions: .pdf, .doc, .jpg, .png'),
+    interest_in_business = models.TextField(
+        blank=True, verbose_name=_('Why are you interested in this business line?'),
```

Comment:
As we submit the form on back button click to save the current data of the user, we need this field to be an optional field in that case (Otherwise user won't be able to go back with an empty interest or background field). However, there are different validations for the next/submit click that essentially make this a mandatory field.